requirements.txt

```diff
@@ -7,4 +7,9 @@ dash-html-components
 
 # The latest version of doit supporting Python 2 is 0.29.0
 doit==0.29.0; python_version < '3.0'
-doit; python_version >= '3.0'
\ No newline at end of file
+doit; python_version >= '3.0'
+
+pytest==4.0.0; python_version < '3.0'
```

Comment:
Why is version pinning necessary for `pytest`?

Reply:
sadly yes, in order to work on all machines 😞 
it seems pytest and pytest-cov need to have matching versions, and the osx machine did not pull the right ones. Only earlier versions work well with python 2

Reply:
Ok. Then let's leave it like this.